_posts/2025-01-31-this-week-in-rails.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Minimal apps, reply_to address, rotate secrets, podman support"
+categories: news
+author: claudiob
+og_image: assets/images/this-week-in-rails.png
+published: true
+date: 2025-01-31
+---
+
+
+Hi, it's [Claudio Baccigalupo](https://github.com/claudiob).
+Let's explore this week’s changes in the Rails codebase.
+
+[Extend --minimal option](https://github.com/rails/rails/pull/54313)
```

Comment:
```suggestion
[Extend --minimal option](https://github.com/rails/rails/pull/54313)  
```

Reply:
The 2 spaces is needed in all the headers to have a line break. You can preview it on jekyll locally